ios/StatusIm.xcodeproj/project.pbxproj

```diff
@@ -542,7 +542,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = "/usr/bin/env sh";
-			shellScript = "set -o errexit\nexport NODE_BINARY=\"${NODE_BINARY:-node}\"\nexport NODE_ARGS=\"${NODE_ARGS:- --max-old-space-size=16384 }\"\n\nbash -x ../node_modules/react-native/scripts/react-native-xcode.sh > ./logs/react-native-xcode.log 2>&1";
+			shellScript = "set -o errexit\nexport NODE_BINARY=\"${NODE_BINARY:-node}\"\nexport NODE_ARGS=\"${NODE_ARGS:- --max-old-space-size=16384 }\"\n\nbash -x ../node_modules/react-native/scripts/react-native-xcode.sh > "\"${PROJECT_DIR}/../logs/react-native-xcode.log\" 2>&1\";
```

Comment:
Unnecessary quote, same in line below:

![image](https://github.com/status-im/status-mobile/assets/2212681/cee4197c-ffdc-42b1-a0cc-817bec3397f5)

Also the last quote is unnecessarily escaped in all lines.